roles/kubernetes/node/tasks/install.yml

```diff
@@ -42,6 +42,28 @@
     - hyperkube
     - upgrade
 
+- name: install | Copy hyperkube binary from download dir
```

Comment:
I guess this part is not required. 
hyperkube binary being moved/renamed to `{{ bin_dir }}/kubelet`

Reply:
`hyperkube` binary includes `cloud-controller-manager`, `kube-apiserver`, `kube-controller-manager`, `kube-proxy`, `kube-scheduler`, `kubectl` and `kubelet` code.
You can call any of the binaries like `hyperkube kubectl` but once you rename `hyperkube` into one of them, you won't be able to call any other binaries (at least I couldn't find a way to do it)

Reply:
Bottom line, I can't use `{{ bin_dir }}/kubelet` to run `kube-proxy --cleanup` in the node main task